### PR TITLE
Adds GitLens Account Management pages

### PR DESCRIPTION
**Pull request by jkelroy**
TODO: add wordpress nav

### CODE REVIEW

File: gitlens/gitlens-edu-pack.md
Comment (GradyGaugler) [inline gitlens/gitlens-edu-pack.md:1]: Is this new or existing? If it is new we should remove it until Vikram provides guidance on how to address EDU as he mentioned earlier.

Comment (GradyGaugler) [inline gitlens/gitlens-edu-pack.md:1]: Is this new or existing content? If new let's remove it and wait for Vikram's instruction on how to handle EDU.

**Review by GradyGaugler — APPROVED** [review]

File: gitlens/gitlens-edu-pack.md
Comment (jkelroy) [reply gitlens/gitlens-edu-pack.md:1]: removed

**Review by GradyGaugler — COMMENTED** [review]
Good to go if theprevious changes were made

**Pull request merged**